subgraph.rinkeby.yaml

```diff
@@ -146,4 +146,29 @@ dataSources:
         - event: TransferWithReferenceAndFee(address,uint256,indexed bytes,uint256,address)
           handler: handleTransferWithReferenceAndFee
       file: ./src/ethConversionProxy.ts
-  
\ No newline at end of file
+  - kind: ethereum/contract
+    name: ERC20EscrowToPay
+    network: rinkeby
+    source:
+      address: "0x8230e703B1c4467A4543422b2cC3284133B9AB5e"
+      abi: ERC20EscrowToPay
+      startBlock: 9669613
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.4
+      language: wasm/assemblyscript
+      entities:
+        - Payment
```

Comment:
Aren't you missing the entity `EscrowEvent` ?

Reply:
Yes definitely, it got overwritten by the prepare script